sdk/identity/identity/src/credentials/managedIdentityCredential/utils.ts

```diff
@@ -72,3 +73,34 @@ export function parseExpiresOn(body: TokenResponseParsedBody): number {
     `Failed to parse token expiration from body. expires_in="${body.expires_in}", expires_on="${body.expires_on}"`
   );
 }
+
+/**
+ * Given a token response, return the expiration timestamp as the number of milliseconds from the Unix epoch.
+ * @param body - A parsed response body from the authentication endpoint.
+ */
+export function getRefreshAt(body: TokenResponseParsedBody): number {
+  if (typeof body.refresh_in === "number") {
+    return Date.now() + body.refresh_in * 1000;
+  } else {
+    const durationInMilliseconds = parseExpiresOn(body) - Date.now();
```

Comment:
Should we rename this internal method so that it's not focused on ExpiresOn, but a general MSAL Date?

Reply:
`parseExpiredOn` returns us the expiration timestamp and it processes the expires_in and expires_on body parameters. Are you suggesting us to rename it to something like `parseExpirationTimestamp` to reflect that?

Reply:
Something like that yes